celery/worker/control.py

```diff
@@ -469,7 +469,7 @@ def memdump(state, samples=10, **kwargs):  # pragma: no cover
 def pool_grow(state, n=1, **kwargs):
     """Grow pool by n processes/threads."""
     if state.consumer.controller.autoscaler:
-        state.consumer.controller.autoscaler.force_scale_up(n)
+        return nok("pool_grow is not supported with autoscale. Adjust autoscale range instead.")
```

Comment:
what does nok actually do?

Reply:
`nok` is just a wrapper:
https://github.com/celery/celery/blob/master/celery/worker/control.py#L38